### ISSUE TITLE
"Link" functionality in text menu causes error

### ISSUE THREAD

**Issue by jancborchardt** [stefan-niedermann/nextcloud-notes#841]
**To Reproduce**
1. Go to a note
2. Mark some text
3. From the formatting menu, choose "Link"

```
App Version: 2.12.1
App Version Code: 2012001
App Flavor: fdroid

Files App Version Code: com.nextcloud.client

---

OS Version: 3.4.0-perf-g284400a(de8920e892)
OS API Level: 27
Device: FP2
Manufacturer: Fairphone
Model (and Product): FP2 (FP2)

---

java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
	at it.niedermann.owncloud.notes.util.ClipboardUtil.getClipboardURLorNull(ClipboardUtil.java:48)
	at it.niedermann.owncloud.notes.util.format.ContextBasedRangeFormattingCallback.onActionItemClicked(ContextBasedRangeFormattingCallback.java:91)
	at android.widget.Editor$TextActionModeCallback.onActionItemClicked(Editor.java:3952)
	at com.android.internal.policy.DecorView$ActionModeCallback2Wrapper.onActionItemClicked(DecorView.java:2459)
	at com.android.internal.view.FloatingActionMode$3.onMenuItemSelected(FloatingActionMode.java:96)
	at com.android.internal.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:761)
	at com.android.internal.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:167)
	at com.android.internal.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:908)
	at com.android.internal.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:898)
	at com.android.internal.view.FloatingActionMode.lambda$-com_android_internal_view_FloatingActionMode_4856(FloatingActionMode.java:121)
	at com.android.internal.view.-$Lambda$IoKM3AcgDw3Ok5aFi0zlym2p3IA.$m$0(Unknown Source:4)
	at com.android.internal.view.-$Lambda$IoKM3AcgDw3Ok5aFi0zlym2p3IA.onMenuItemClick(Unknown Source:0)
	at com.android.internal.widget.FloatingToolbar$FloatingToolbarPopup$2.onClick(FloatingToolbar.java:425)
	at android.view.View.performClick(View.java:6294)
	at android.view.View$PerformClick.run(View.java:24770)
	at android.os.Handler.handleCallback(Handler.java:790)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6494)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Comment by stefan-niedermann** [comment]
Fixed and will be released with next version. Thanks for report.

**Comment by jancborchardt** [comment]
Ah, thanks! Sorry if I missed a duplicate, was on mobile

**Comment by stefan-niedermann** [comment]
Nope, i just fixed it^^

**Comment by jancborchardt** [comment]
Ah, Github mobile doesn’t seem to show the `stefan-niedermann closed this in 81f71ca 3 hours ago` notice yet. :)
